ci/params.py

```diff
@@ -84,6 +89,11 @@ class AllParams:
         description='config name of the key to use for signing the packages',
         default='gardenlinux',
     )
+    manifest_set_key = NamedParam(
+        name='manifest_set_key',
+        default='~',
```

Comment:
nit: `None` (or `null`) might be a bit more intuitive. If the value is to be translated to YAML, the alternatives I gave are equivalent

Reply:
Done